### PR TITLE
test: e2e refactors [TESTENG-2]

### PR DESCRIPTION
**Pull request by JComins000**
## Ticket

[TESTENG-2]

## Description

refactors some common functionality into base models

## Checklist

- [ ] Changes have been manually QA'd
- [ ] New features have been approved by the corresponding PM
- [ ] User-facing API changes have the "User-facing API Change" label
- [ ] Release notes have been added as a separate file under `docs/release-notes/`
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses have been included for new code which was copied and/or modified from any external code

[TESTENG-2]: https://hpe-aiatscale.atlassian.net/browse/TESTENG-2?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DET-123]: https://hpe-aiatscale.atlassian.net/browse/DET-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: webui/react/src/e2e/models/pages/Admin/UserManagement.ts [hunk 1]
```diff
@@ -38,7 +38,7 @@ export class UserManagement extends AdminPage {
     selector: '[data-testid="statusSelect"]',
   });
   readonly actions = new ActionDropdownMenu({
-    childNode: new BaseComponent({
+    clickThisComponentToOpen: new BaseComponent({
```
Comment (djanicekpach) [inline webui/react/src/e2e/models/pages/Admin/UserManagement.ts:41]: I like this name a lot better!

File: webui/react/src/e2e/models/components/Page.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { BaseReactFragment } from 'e2e/models/BaseComponent';
+import { BaseReactFragment } from 'e2e/models/base/BaseReactFragment';
 
 /**
  * Represents the PageComponent component in src/components/Page.tsx
```
Comment (djanicekpach) [inline webui/react/src/e2e/models/components/Page.ts:4]: Could this `PageComponent` file be in a `common` or `base` folder too?

**Review by djanicekpach — CHANGES_REQUESTED** [review]
This looks good. Could we even go a step further though? I'm thinking ahead a bit so some details aren't clear, but what do you think about having a `common` folder housing `base`, `ant`, and `hew`? It would be nice if when another repo comes and asks about using UI tests, we could just lift the common folder into it's own repo.

File: webui/react/src/e2e/models/components/Page.ts (hunk 2 above)
Comment (JComins000) [reply webui/react/src/e2e/models/components/Page.ts:4]: nope. it's a model that represents the component from https://github.com/determined-ai/determined/blob/main/webui/react/src/components/Page.tsx

Comment (JComins000) [reply webui/react/src/e2e/models/components/Page.ts:4]: since they have a Page.tsx, we have a Page.ts model.

**Comment by JComins000** [comment]
@djanicekpach
> This looks good. Could we even go a step further though? I'm thinking ahead a bit so some details aren't clear, but what do you think about having a common folder housing base, ant, and hew?

Good idea. I think base ant and hew will all eventually live in node modules-- but for now they can live in `common`

**Review by djanicekpach — APPROVED** [review]

File: webui/react/src/e2e/models/ant/Dropdown.ts
Comment (JComins000) [inline webui/react/src/e2e/models/ant/Dropdown.ts:1]: this isnt actually deleted, just moved

**Pull request merged**